### PR TITLE
Cart view base

### PR DESCRIPTION
**Pull request by dom-arm**
The cart view i functional. However you cannot update or remove an item.
By "/cart/<cart_id>" you can reach the cart (if there is one in the db, otherwise the url does not exist)
I also fixed the warning of the created_at field so it should be gone now.

### CODE REVIEW

File: cart/migrations/0004_merge_0002_initial_0003_alter_cart_created_at.py
Comment (milk-mode) [inline cart/migrations/0004_merge_0002_initial_0003_alter_cart_created_at.py:1]: Same as above :)

File: cart/migrations/0005_alter_cart_created_at.py
Comment (milk-mode) [inline cart/migrations/0005_alter_cart_created_at.py:1]: Same as above :)

File: cart/migrations/0008_alter_cart_created_at.py
Comment (milk-mode) [inline cart/migrations/0008_alter_cart_created_at.py:1]: Same as above :)

File: cart/migrations/0001_initial.py
[the remark this answers is not in the document]
Comment (dom-arm) [reply cart/migrations/0001_initial.py:1]: Ofc, thank you! I will delete them now!

File: product/migrations/0001_initial.py
Comment (milk-mode) [inline product/migrations/0001_initial.py:1]: Same as above :)

**Comment by dom-arm** [comment]
Now I think everything will be correct. I removed all the migration files in this pull request. And when the new .gitignore is added to main we can merge this pull request (if it looks good to you). It will be a conflict in .gitignore when merging but we just keep the changes in main.

**Review by milk-mode — APPROVED** [review]

**Pull request merged**